core/src/main/java/com/nerds/gamejam/ecs/system/MonsterControlSystem.java

```diff
@@ -0,0 +1,47 @@
+package com.nerds.gamejam.ecs.system;
+
+import com.artemis.Aspect;
+import com.artemis.ComponentMapper;
+import com.artemis.annotations.Wire;
+import com.artemis.systems.IteratingSystem;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Input;
+import com.nerds.gamejam.ecs.component.MonsterComponent;
+import com.nerds.gamejam.ecs.component.PositionComponent;
+import com.nerds.gamejam.ecs.component.VelocityComponent;
+import com.nerds.gamejam.gameplay.character.Monster;
+
+@Wire
+public class MonsterControlSystem extends IteratingSystem {
```

Comment:
I'd probably split this class out. Add a generic add velocity to position System, so anything with those components will automatically be able to move. Then add a MonsterControlSystem and all it does is add the Velocity component to the monster until it reaches the next stage, then it removes the Velocity component